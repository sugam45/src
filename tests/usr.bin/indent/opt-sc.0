/* $NetBSD: opt-sc.0,v 1.2 2021/03/07 10:12:18 rillig Exp $ */
/* $FreeBSD$ */

/* comment
without
asterisks
*/

/*
** This comment style is used by Lua.
*/

/**
 * Javadoc, adopted by several other programming languages.
 */
