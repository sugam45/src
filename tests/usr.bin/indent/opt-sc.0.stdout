/* $NetBSD: opt-sc.0.stdout,v 1.2 2021/03/07 10:12:18 rillig Exp $ */
/* $FreeBSD$ */

/*
 * comment without asterisks
 */

/* $ XXX: The additional '*' is debatable. */
/*
 * * This comment style is used by Lua.
 */

/**
 * Javadoc, adopted by several other programming languages.
 */
